parts/k8s/kubernetesconfigs.sh

```diff
@@ -259,6 +258,10 @@ function ensureKubelet() {
     KUBELET_RUNTIME_CONFIG_SCRIPT_FILE=/opt/azure/containers/kubelet.sh
     wait_for_file 1200 1 $KUBELET_RUNTIME_CONFIG_SCRIPT_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     systemctlEnableAndStart kubelet
+    # Don't start the kubelet-health-monitor systemd unit (yet).
+    # KUBELET_HEALTH_MONITOR_SYSTEMD_FILE=/etc/systemd/system/kubelet-health-monitor.service
+    # wait_for_file 1200 1 $KUBELET_HEALTH_MONITOR_SYSTEMD_FILE || exit $ERR_FILE_WATCH_TIMEOUT
+    # systemctlEnableAndStart kubelet-health-monitor
```

Comment:
what's the plan here?

Reply:
The health-monitor.sh script also offers kubelet health monitoring, so I enabled it and tested that it works. But @jackfrancis suggested we take it slow by not adding this yet, which I can’t argue with.

I commented out these lines so it would be simple to turn on the kubelet monitoring if and when we choose to do so. But I’m happy just to remove all that from this PR if we prefer.

Reply:
I'm fine with the commented out code, just wanted to know the reasoning behind it